src/components/card/video-card/video-card.jsx

```diff
@@ -45,18 +35,21 @@ const VideoCard = ({videoDetails}) => {
                     <div>{`${creator}`}</div>
```

Comment:
`<p className="card-header">{title.length>60?`${title.substring(0,48)}...`:title}</p>`

Like the way, you maintained string length. there is a CSS-only method also which specifies how many lines it should show before adding ...
```
-webkit-box-orient: vertical;
-webkit-line-clamp: 2;
```

Reply:
Thanks for telling me this. I have made the changes. Thanks for reviewing.